rossum_api/api_client.py

```diff
@@ -316,6 +316,12 @@ async def request_json(self, method: str, *args, **kwargs) -> Dict[str, Any]:
             return {}
         return response.json()
 
+    async def get_token(self, refresh: bool = False) -> str:
+        """Returns the current token. Authenticate if needed."""
```

Comment:
I would suggest smt like:

```
Returns the current token. Authentication is done automatically if needed.

Arguments:
----------
refresh
    force refreshing the token
```
The same doc string could be added to the corresponding method in other modules.

Reply:
Looks better. Thanks :)